docs/developers/docs.rst

```diff
@@ -0,0 +1,47 @@
+Building the Docs Locally
+=========================
+
+The Hypothesis client's documentation is hosted at https://h.readthedocs.io/projects/client/.
+The docs are built using https://readthedocs.org.
```

Comment:
```suggestion
The docs are built using [Sphinx](https://docs.readthedocs.io/en/stable/intro/getting-started-with-sphinx.html) and hosted by https://readthedocs.org.
```

Reply:
I'd like to leave this as just a link to RTD's home page, which includes a link to their docs. Technically it's not just Sphinx that our docs use, it's the RTD system, which has its own docs and adds various particular ways of doing things and additional features on top of Sphinx. Markdown and MkDocs support, for example, and GitHub integration, canonical URLs, versioning, their own configuration file format, their dashboard, custom domain support, sub projects, redirects, localisation, etc. The RTD docs link to the reST and Sphinx docs as appropriate, as well as providing their own "Getting started with Sphinx" guide. So I think readthedocs.org is the correct starting page for "How do our docs work?"